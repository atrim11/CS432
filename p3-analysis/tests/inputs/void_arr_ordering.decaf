int arr[5];
void arr2[3];
void arr3[0];
int arr4[1];
int arr5[0];
