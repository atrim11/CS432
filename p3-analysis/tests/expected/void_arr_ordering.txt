Void variable 'arr2' on line 2
Void variable 'arr3' on line 3
Array 'arr3' on line 3 must have positive non-zero length
Array 'arr5' on line 5 must have positive non-zero length
Program does not contain a 'main' function
